portal-frontend/src/app/features/view-application/view-application.component.html

```diff
@@ -1,68 +1,55 @@
-<div class='buttons' *ngIf='application'>
+<div class="buttons" *ngIf="application">
   <div>
-    <button *ngIf='application.canEdit' mat-stroked-button color='warn' (click)='onCancel(application.fileNumber)'>
+    <button *ngIf="application.canEdit" mat-stroked-button color="warn" (click)="onCancel(application.fileNumber)">
       Cancel Application
     </button>
   </div>
   <div>
     <button
       [routerLink]="'/application/' + application.fileNumber + '/edit'"
-      *ngIf='application.canEdit'
+      *ngIf="application.canEdit"
       mat-flat-button
-      color='primary'
+      color="primary"
     >
       Edit Application
     </button>
-    <button *ngIf='application.canReview' mat-flat-button color='primary'>Review Application</button>
+    <button *ngIf="application.canReview" (click)="onReview(application.fileNumber)" mat-flat-button color="primary">
+      Review Application
+    </button>
   </div>
 </div>
-<div *ngIf='application' class='content'>
+<div *ngIf="application" class="content">
   <section>
     <h5>1. Application</h5>
-    <div class='view-box'>
```

Comment:
I wonder what is the difference here from other files, since I noticed that single quotes got replaced with double quotes? 🤔

Reply:
My IDE is being a bit odd and if I dont explicitly run prettier on HTML files it uses the IDE formatter instead which is single quotes. I'm trying to make sure to run prettier on each HTML file before I commit though